worksheet_B/Mandelbrot.cpp

```diff
@@ -18,18 +18,61 @@ int main()
 	for (int pixelY = 0; pixelY < image.height(); pixelY++)
 	{
 		// TODO: Map the y coordinate into the range minY to maxY
-		//double y0 =
+		double y0 = ((pixelY / 800.0) * (maxY - minY) + minY);
 
 		for (int pixelX = 0; pixelX < image.width(); pixelX++)
 		{
 			// TODO: Map the x coordinate into the range minX to maxX
-			//double x0 =
+			double x0 = ((pixelX / 800.0) * (maxX - minX) + minX);
 
 			// TODO: implement the algorithm to colour a single pixel (x0, y0) of the Mandelbrot set fractal
 			// The code below simply fills the screen with random pixels
+			double x = 0.0;
+			double y = 0.0;
+			int iteration = 0.0;
+			int MaxIteration = 200;
 
 			// Write the pixel
 			// TODO: change the right-hand side of these three lines to write the desired pixel colour value
+			int red = 0;
+			int blue = 0;
+			int green = 0;
+			
+			if (iterations <= 0)
```

Comment:
This doesn't compile because `iterations` doesn't exist. Did you mean `iteration` (no S on the end)?